src/main/java/io/pinecone/utils/Constants.java

```diff
@@ -0,0 +1,11 @@
+package io.pinecone.utils;
+
+public class Constants {
```

Comment:
IMO these constants should all probably live with the `PineconeIndexOperationClient`. I don't see any benefit or removing these values from that context and putting them in a different class because they're not being used elsewhere. `Constants` is kind of a vague term that over time can end up collecting a lot of unrelated stuff in it.

Reply:
Good point, I'll move them within PineconeIndexOperationClient class.